test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -376,7 +376,7 @@ private void CategoricalFeaturizationOn(params string[] dataPath)
             );
 
             // Read the data.
-            var data = reader.Read(dataPath);
+            var data = reader.Read(new MultiFileSource(dataPath));
```

Comment:
>MultiFileSource [](start = 39, length = 15)

I don't see why just the string dataPath is not sufficient?

Reply:
Is the extension of #1281 not being picked up for some reason? We should continue to use it. It makes the sample so much more pleasant to read. Back before we had that, it was something that came up perennially from the .NET team, users, our own developers, so it would be a shame if we had somehow regressed on that.

Reply:
I had to add the method `Read(params string[] path)`  in TextLoader.cs  to get  it working.

The  regression in this PR is because `TextLoaderStatic` used to be a partial class `TextLoader`


---
In reply to: [243578060](https://github.com/dotnet/machinelearning/pull/1930#discussion_r243578060) [](ancestors = 243578060)

Reply:
Had to fix a bug in TextLoader.cs  to get it to work.

---
In reply to: [243471085](https://github.com/dotnet/machinelearning/pull/1930#discussion_r243471085) [](ancestors = 243471085)